mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+                  required: true
+                  description: |-
+                    URL of the Container image in Google Container Registry or Google Artifact Registry. More info: https://kubernetes.io/docs/concepts/containers/images
+                - !ruby/object:Api::Type::Array
+                  name: "command"
+                  description: |-
+                    Entrypoint array. Not executed within a shell. The docker image's ENTRYPOINT is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. More info: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#running-a-command-in-a-shell
+                  item_type: Api::Type::String
+                - !ruby/object:Api::Type::Array
+                  name: "args"
+                  description: |-
+                    Arguments to the entrypoint. The docker image's CMD is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. More info: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#running-a-command-in-a-shell
+                  item_type: Api::Type::String
+                - !ruby/object:Api::Type::Array
+                  name: "env"
+                  description: |-
+                    List of environment variables to set in the container.
+                  item_type: !ruby/object:Api::Type::NestedObject
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: "name"
+                        required: true
+                        description: |-
+                          Name of the environment variable. Must be a C_IDENTIFIER, and mnay not exceed 32768 characters.
+                      - !ruby/object:Api::Type::String
+                        name: "value"
+                        description: |-
+                          Variable references $(VAR_NAME) are expanded using the previous defined environment variables in the container and any route environment variables. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Defaults to "", and the maximum length is 32768 bytes
+                        # exactly_one_of:
```

Comment:
`ConflictsWith`, `ExactlyOneOf` and `AtLeastOneOf` within lists are currently not supported at SKD level.